test/index.js

```diff
@@ -164,9 +183,11 @@ describe('Navigation', () => {
     await server.register(register);
     await server.initialize();
 
-    const res = await server.inject({ url: '/graphql', method: 'post', payload: { query: 'query { account { services { name, url } } }' } });
+    const res = await server.inject({ url: '/graphql', method: 'post', payload: { query: 'query { account { services { name, slug, url } } }' } });
     expect(res.result.data.account.services[0].name).to.equal('Logout');
     expect(res.result.data.account.services[0].url).to.equal('/logout');
+    expect(res.result.data.account.services[1].slug).to.equal('change-password');
+    expect(res.result.data.account.services[1].url).to.equal('https://sso.joyent.com/login/changepassword/4fc13ac6-1e7d-cd79-f3d2-96276af0d638');
```

Comment:
I don't believe this is the actual URL in sdcsso (the `/login` wouldn't be there).

Reply:
nice catch

Reply:
fixed